lib/conmonmon.go

```diff
@@ -0,0 +1,177 @@
+package lib
+
+import (
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/containers/psgo"
+	"github.com/cri-o/cri-o/oci"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+)
+
+// the time between checking the registered conmons
+var sleepTime = 5 * time.Minute
```

Comment:
Is this something that should be configurable via the conf file?

Reply:
that's one of the things I'd wire up if there's the ask